### PR TITLE
Unity v4

### PR DESCRIPTION
**Pull request by crimsonred**

### CODE REVIEW

File: PubNubUnity/Assets/PubNubUnity/PubNubUnityBase.cs [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+            #elif(UNITY_WEBGL)
+            Version = string.Format("PubNub-CSharp-UnityWebGL/{0}", build);
+            #else
+            Version = string.Format("PubNub-CSharp-Unity/{0}", build);
+            #endif
+            #if (ENABLE_PUBNUB_LOGGING)
+            this.PNLog.WriteToLog (Version, PNLoggingMethod.LevelInfo);
+            #endif
+
+            if (GameObjectRef == null) {
```
Comment (schakkis) [inline PubNubUnity/Assets/PubNubUnity/PubNubUnityBase.cs]: I'm guessing this is supposed to be gameObjectRef and that GameObjectRef should be assigned in the else-clause?

File: PubNubUnity/Assets/EndPoints/PubSub/SubscribeBuilder.cs [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        public void Execute(){
+            pubBuilder.Execute();
+        }
+
+        public SubscribeBuilder WithPresence(){
+            pubBuilder.WithPresence();
+            return this;
+        }
+
+        public SubscribeBuilder SetChannels(List<string> channels){
```
Comment (schakkis) [inline PubNubUnity/Assets/EndPoints/PubSub/SubscribeBuilder.cs]: Most of the other builders seem to use Channels (and other setters) without the Set-prefix.

Comment (crimsonred) [reply PubNubUnity/Assets/EndPoints/PubSub/SubscribeBuilder.cs]: thats true, modifying. Thanks @schakkis !

File: PubNubUnity/Assets/PubNubUnity/PubNubUnityBase.cs (hunk 1 above)
Comment (crimsonred) [reply PubNubUnity/Assets/PubNubUnity/PubNubUnityBase.cs]: thats also true, modified. Thanks @schakkis !

**Pull request merged**